build.gradle

```diff
@@ -219,6 +219,7 @@ task('format') {
     dependsOn generate
     dependsOn spotlessApply
     dependsOn subprojects.collect { it.getTasksByName('blackFormat', true) }
+    dependsOn subprojects.collect { it.getTasksByName('isortFormat', true) }
```

Comment:
does it make sense to add flake/black/eetc.. all python formatters/linters to a `pythonFormat` task then just depend on `pythonFormat` here?